src/neighborsearch/neighborhood.jl

```diff
@@ -54,8 +55,11 @@ end
 
 # search method for ball neighborhood
 function search(xₒ::AbstractVector, method::NeighborhoodSearch{O,N,T};
-                mask=nothing) where {O,N<:BallNeighborhood,T}
-  inds = inrange(method.tree, xₒ, radius(method.neigh))
+                mask=nothing) where {O,N<:AbstractBallNeighborhood,T}
+  neigh = method.neigh
+  range = N <: BallNeighborhood ? radius(neigh) : one(eltype(neigh.semiaxes))
```

Comment:
We need provide a public API to recover this "one" for the ellipsoid case. Or perhaps we need to modify the original BallNeighborhood so that the "one" can be used too without the need to querying the `radius`. Any ideas on how we should proceed here? At the moment I don't have a good name for this public API function that returns 1 in the case of ellipsoid and `radius(neigh)` in the case of ball.